src/utils/autorestOptions.ts

```diff
@@ -61,6 +63,11 @@ async function getHeadAsBoolean(host: Host): Promise<boolean> {
   return Boolean(headAsBoolean);
 }
 
+async function getIsTestPackage(host: Host): Promise<boolean> {
+  const isTestPackage: boolean = await host.GetValue("is-test-package");
```

Comment:
same comment about the name, `is-released-package`

Reply:
Here the is-test-package looks fine